src/Automation/ScannerFactory.cs

```diff
@@ -16,9 +16,9 @@ public static class ScannerFactory
         /// <returns></returns>
         public static IScanner CreateScanner(Config config)
         {
-            return new Scanner(config
-                , Factory.CreateOutputFileHelper(config.OutputDirectory)
-                , new ScanResultsAssembler());
+            // Factory.CreateOutputFileHelper(config.OutputDirectory)
```

Comment:
Should line 19 be removed, rather than commented out?

Reply:
It has been taken out in an upcoming PR.